packages/headless/src/features/facets/facet-set/interfaces/options.ts

```diff
@@ -1,6 +1,8 @@
-import {FacetRequest} from './request';
+import {FacetOptions} from '../../../../product-listing.index';
 
-export type FacetOptionalParameters = Pick<
-  FacetRequest,
-  'filterFacetCount' | 'injectionDepth' | 'numberOfValues' | 'sortCriteria'
+export type FacetOptionalParameters = Required<
```

Comment:
The `sortCriteria` in the FacetRequest interface can now either be `FacetSortCriterion` OR `SpecificSortCriteriaExplicitAlphanumeric`, which was leading to type problems that I couldn't resolve in a few places, so I had to update this type such that it remained the same as before.